### ISSUE TITLE
how to generate motions with labels specified by users?

### ISSUE THREAD

**Issue by wang-zm18** [TaeryungLee/MultiAct_RELEASE#1]
I have tried to change the target_labels in gen.yaml， however, it occurs the compatibility problem with "stats.pkl". Thank you in advance!

**Comment by TaeryungLee** [comment]
Hi, you can use the labels specified in list "label_over_twenty" in data/datautils/babel_label.py.

**Comment by wang-zm18** [comment]
OK, thank you! I will have a try!


Taeryung Lee ***@***.***> 于2023年3月11日周六 14:36写道：

**Comment by wang-zm18** [comment]
BTW, does the model support arbitrary  specified action  sequences?

**Comment by TaeryungLee** [comment]
Yes, our model supports the arbitrary specified action sequences.
However, we have tested our model using the sequence included in the test set of BABEL.
The generation result from too irrelevant action sequences could be not so realistic.

**Comment by wang-zm18** [comment]
Thank you, Mr Lee. If I use the labels in  "label_over_twenty" in
babel_label.py, it did work!  However,  if I use arbitrary artificial
actions  made by myself, it still failed. So how to fix this problem? I
just want to see the generalization of the pretrained model. Thank you in
advance!

Taeryung Lee ***@***.***> 于2023年3月14日周二 16:44写道：
